pom.xml

```diff
@@ -182,7 +182,7 @@
     <kryo.version>3.0.3</kryo.version>
     <libfb303.version>0.9.3</libfb303.version>
     <libthrift.version>0.9.3</libthrift.version>
-    <log4j2.version>2.10.0</log4j2.version>
+    <log4j2.version>2.15.0</log4j2.version>
```

Comment:
2.16 here

Reply:
while it is not required to have 2.16, might as well get it done.